### DISCUSSION TITLE
화면 이동 관련 질문이 있습니다.

### DISCUSSION THREAD

**Discussion by xenossys** [nsoojin/MiniSuperApp-fastcampus#10] [아무 질문이나 환영합니다]
선생님 저는 원래 스토리보드로 개발을 진행하고 있었는데요.

스토리 보드에서는 화면을 이동하면 swipe로 뒤로가기가 자동으로 되어서 별로 신경쓰던 기능이 아니었는데

본 강의에서 홈 > 슈퍼마트 버튼을 클릭해서 이동한 화면에서 swipe로 뒤로가기가 안되더라구요.

어떠한 설정을 해줘야 swipe back이 가능할까요?

**Comment by nsoojin** [comment]
평범하게 UINavigationController로 푸시 하셨으면 잘 돼야할거 같은데요, 어떻게 이동하셨는지 코드를 봐야 알거 같습니다.

**Reply by nsoojin** [reply]
슈퍼택시 화면은 내비게이션컨트롤러를 쓰지 않고 push처럼 보이는 커스텀 애니메이션을 넣은 모달 presentation입니다. 내비게이션컨트롤러처럼 interactive pop을 구현하고 싶으시면 커스텀 애니메이션을 구현하셔야합니다.
